build-docs.js

```diff
@@ -38,14 +45,7 @@ async function build(currentDir, tmpl) {
       let buffer = await readFile(filename);
```

Comment:
I think we should add `utf8` here to work with the string instead of the buffer..

That can be a future PR 😄